### ISSUE TITLE
(Unraid) Env variables do not work

### ISSUE THREAD

**Issue by ocgineer** [Kareadita/KavitaEmail#11]
When using KavitaEmail docker on Unraid it seems that the Docker Env variables do not seem to work.

The user has to delete the env variables, mount a config folder, and copy the default config folder from github and adjust the settings in the  config file. This resolves the issue but it is not out of the box using the env variables.

using the env variables and then trying to test the connection results in error 500 internal server error.

![image](https://user-images.githubusercontent.com/12483966/189495367-a6697098-c4d5-4bdd-a173-c5cedb40f8d0.png)

Original post in Discord;

it seems to be indeed the env variables passed to the app not working correctly (at least on the Unraid docker image, cant test other docker systems) or something else goes wrong. I removed all env vars, mounted `/app/config:/mnt/user/appdata/kavita-email`  on first startup it threw an error that it couldn't find settings json, so I copied it over from the windows app I used to test earlier and Kavita connects now successfully.

On an forgot password attempt the console showed also that it couldn't find the templates, so I had to make them myself from the GitHub code, they were not included in the windows or Linux release on GitHub (tar.gz) but it looks like the docker image copies over the template folder to /app/config/templates so that shouldn't the the issue normally. i think by mounting it just cleared config folder.

After thus removing the env vars, mounting the config file, creating manually the templates and connecting Kavita to http://kavita-email:5003 it works at least, so just need to resolve this for deployment.

**Comment by ocgineer** [comment]
I looked further into this and it seems that the person that added KavitaEmail to the Community Apps forgot to set the Key name (for each) of the environment variables. So only this needs to be adjusted to be the default so Unraid users that install KavitaEmail via the Community Apps do not have to do this manually. I have verified that after correcting this, appsettings.json is correctly populated with the set environment variables.

![image](https://user-images.githubusercontent.com/12483966/192071907-6d26dca9-4c97-4908-b1e3-16ec0f3018e9.png)

Which needs to be

![image](https://user-images.githubusercontent.com/12483966/192071953-209909ed-abf4-4c4e-b3d9-61311b2acca2.png)
